Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -37,6 +38,29 @@ extension BrowserViewController {
         popover.addsConvenientDismissalMargins = false
         popover.present(from: topToolbar.locationView.rewardsButton, on: self)
     }
+    
+    func configureRewardsLogs() {
+        if rewards == nil { return }
```

Comment:
This should be actually be setup before `rewards` is created so that logs come in during initialization